src/e-build.js

```diff
@@ -33,7 +34,37 @@ function runNinja(config, target, ninjaArgs) {
   const exec = os.platform === 'win32' ? 'ninja.exe' : 'ninja';
   const args = [...ninjaArgs, target];
   const opts = { cwd: evmConfig.outDir(config) };
+
+  const start = Date.now();
   depot.execFileSync(config, exec, args, opts);
+  recordBuildTiming({
+    elapsed_time_ms: Date.now() - start,
+    build_target: target,
+    electron_version: readElectronVersion(config),
+  });
+}
+
+function recordBuildTiming({ elapsed_time_ms, build_target, electron_version }) {
+  if (process.env.EVM_SKIP_TELEMETRY) return;
+  // If it fails, nbd, we'll just miss this timing.
+  const req = https.request({
+    hostname: 'electron-build-perf-21c0b.firebaseio.com',
```

Comment:
For multiple reasons this should go through an endpoint in electron/data and get piped into our grafana instance